### ISSUE TITLE
Admin, new document_general, missing constants

### ISSUE THREAD

**Issue by lat9** [zencart/zencart#2564]
zc156b.  Navigate to the admin's _Catalog::Categories/Products_ and create a new product of type _Document - General_.  A log similar to the following will be generated for missing constants:
```
[13-Jul-2019 08:48:00 America/New_York] Request URI: /test_store/adminzc156/product.php?action=new_product, IP address: ::1
#1  require(C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php) called at [C:\xampp\htdocs\test_store\adminzc156\product.php:101]
--> PHP Notice: Use of undefined constant DEFAULT_DOCUMENT_GENERAL_PRODUCTS_VIRTUAL - assumed 'DEFAULT_DOCUMENT_GENERAL_PRODUCTS_VIRTUAL' in C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php on line 21.

[13-Jul-2019 08:48:00 America/New_York] Request URI: /test_store/adminzc156/product.php?action=new_product, IP address: ::1
#1  require(C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php) called at [C:\xampp\htdocs\test_store\adminzc156\product.php:101]
--> PHP Notice: Use of undefined constant DEFAULT_DOCUMENT_GENERAL_TAX_CLASS_ID - assumed 'DEFAULT_DOCUMENT_GENERAL_TAX_CLASS_ID' in C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php on line 27.

[13-Jul-2019 08:48:00 America/New_York] Request URI: /test_store/adminzc156/product.php?action=new_product, IP address: ::1
#1  require(C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php) called at [C:\xampp\htdocs\test_store\adminzc156\product.php:101]
--> PHP Notice: Use of undefined constant DEFAULT_DOCUMENT_GENERAL_PRODUCTS_IS_ALWAYS_FREE_SHIPPING - assumed 'DEFAULT_DOCUMENT_GENERAL_PRODUCTS_IS_ALWAYS_FREE_SHIPPING' in C:\xampp\htdocs\test_store\adminzc156\includes\modules\document_general\collect_info.php on line 35.
```

**Comment by lat9** [comment]
Noting that this occurred on a site being upgraded from zc154.

**Comment by scottcwilson** [comment]
weird.  Missing from db for a while.  PR Coming up.

**Comment by scottcwilson** [comment]
Fixed in #2566.  They were in db, just spelled differently.

**Comment by scottcwilson** [comment]
@lat9 Please close.
